pkg/controller/remotemachineset/vsphereactuator.go

```diff
@@ -0,0 +1,74 @@
+package remotemachineset
+
+import (
+	"fmt"
+
+	machineapi "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
```

Comment:
Combine this import with the other "github.com/openshift" imports that are not "github.com/openshift/hive".